jekyll/_cci2_ja/language-javascript.md

```diff
@@ -226,6 +226,6 @@ workflows:
 ## 関連項目
 {: #see-also-new }
 
-- [Node アプリの Heroku への継続的デプロイ]({{site.blog_base_url}}/ja/continuous-deployment-to-heroku/)
-- [Node.js の Azure VM への継続的デプロイ]({{site.blog_base_url}}/ja/cd-azure-vm/)
+- [Node アプリの Heroku への継続的デプロイ (英語)]({{site.blog_base_url}}/continuous-deployment-to-heroku/)
```

Comment:
```suggestion
- [Node アプリの Heroku への継続的デプロイ](https://circleci.com/ja/blog/continuous-deployment-to-heroku/)
```

Reply:
@Mayuko-Sa Done! I see what the issue before was. In the URL to the blog post, ` ja/blog/` was switched to be `blog/ja/`